kvbc/src/pruning_handler.cpp

```diff
@@ -266,6 +267,14 @@ void PruningHandler::pruneThroughBlockId(kvbc::BlockId block_id) const {
         LOG_FATAL(logger_, "Error while running pruning");
         std::terminate();
       }
+      // Now that we done with pruning, lets run a manual compaction
+      if (manual_compaction) {
+        LOG_INFO(logger_, "start compaction process");
+        auto status = blocks_deleter_.reclaimDiskSpace();
+        if (!status.isOK()) {
+          LOG_ERROR(logger_, "unable to run compaction " << status.toString());
+        }
+      }
```

Comment:
Will this work with a different pruning handler (i.e. the one used in Concord)?

Reply:
yes, but we added this change on the bump also